ime/app/src/main/res/values-en/strings.xml

```diff
@@ -485,6 +483,8 @@
     <string name="bottom_generic_row_dialog_title">Select row type</string>
     <string name="bottom_generic_row_summary">Selected bottom row: %s</string>
 
+    <string name="popup_characters_order_title">Popup characters order</string>
```

Comment:
you can remove these changes in the `en` locale, the github bot will fill those correctly